.gitattributes

```diff
@@ -9,4 +9,3 @@
 *.cc text=auto eol=lf
 *.md text=auto eol=lf
 *.csv text=auto eol=lf
-* text=auto eol=lf
```

Comment:
git really didn't like this, assumed *all* files were text and it would convert the "line endings" in png files, corrupting them.